tests/test_data/0_output.json

```diff
@@ -0,0 +1,169 @@
+{
+    "hostname": "7a3725255f7c",
+    "1": {
+        "load": {
+            "block1": [
```

Comment:
What is block? is it step? I prefer we use same terminology here. So that it's intuitive.

Reply:
Yes, this came up by Loic. Block composed by a set of training steps that do not contain check pointing. 

I don't have another word to replace that yet. So I just keep whatever it is now.